tests/fixtures/flask.py

```diff
@@ -0,0 +1,7 @@
+from webapp.app import app
```

Comment:
I would maybe put this in `tests/client.py`. Thoughts @tbille?

Reply:
I'm trying to replicate the same structure as the example @nottrobin give me:
https://github.com/canonical-web-and-design/canonicalwebteam.yaml-responses/blob/master/tests/test_flask.py

But I'm not sure if this is an good case to use test fixtures. Please let me know to maintain the same structure in all projects. @tbille ?

Reply:
yeah I like the test/client.py

because it feels that fixtures/flask.Py implies that there could be other clients.